org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -571,6 +571,12 @@ private static List<String> getGradleInitScriptArgs() {
 			addInitScriptToArgs(protobufInitScript, args);
 		}
 
+		// Add init script of android support
+		if (preferencesManager != null && preferencesManager.getPreferences().isAndroidSupportEnabled()) {
```

Comment:
we can do the null check at line 569 and early return if the `preferenceManager` is null.